actors/Weapons/Slot1/SAW.dec

```diff
@@ -2048,7 +2048,7 @@ Actor Chain_saw : PB_Weapon //replaces Chainsaw
 		C9S2 LMN 1 A_SetRoll(roll-1,SPF_INTERPOLATE)
 		C9S2 OP 1
 		C9S2 QR 1 A_DoPBWeaponAction(WRF_NOFIRE|WRF_NOSWITCH)
```

Comment:
Comment out or delete A_DoPBWeaponAction(WRF_NOFIRE|WRF_NOSWITCH) after C9S2 QR 1 to prevent an issue that can prevent PB from starting up. To comment out just type // before that piece I mentioned, or delete it if its easier.